sessionctx/variable/session.go

```diff
@@ -109,6 +109,31 @@ func (tc *TransactionContext) ClearDelta() {
 	tc.TableDeltaMap = nil
 }
 
+// InsertValuesBufs is buffers used by insert statement.
```

Comment:
Actually, the buffers are not only used in the insert statement. But also in  delete/update/replace statements.